cmd/kube-apiserver/app/options/validation.go

```diff
@@ -127,7 +127,7 @@ func validateAPIPriorityAndFairness(options *ServerRunOptions) []error {
 		// If none of the following runtime config options are specified, APF is
 		// assumed to be turned on.
 		enabledAPIString := options.APIEnablement.RuntimeConfig.String()
-		testConfigs := []string{"flowcontrol.apiserver.k8s.io/v1beta2", "flowcontrol.apiserver.k8s.io/v1beta1", "api/beta", "api/all"} // in the order of precedence
+		testConfigs := []string{"flowcontrol.apiserver.k8s.io/v1beta3", "flowcontrol.apiserver.k8s.io/v1beta2", "flowcontrol.apiserver.k8s.io/v1beta1", "api/beta", "api/all"} // in the order of precedence
```

Comment:
since the internal controller uses v1beta3, I think we need that specific version enabled, and should drop v1beta2 and v1beta1 from this list

Reply:
for 1.26, we are setting `v1beta2` as the storage version, we should not allow the user to disable `v1beta2` when APF is enabled, or is it orthogonal to the storage version?

Reply:
this controls what gets served, not stored

Reply:
done, I have added a unit test for that as well.